assets/src/scss/layout/_gravity-forms.scss

```diff
@@ -0,0 +1,164 @@
+.gform_wrapper {
+  font-family: $roboto;
+
+  &.gform_validation_error .gform_validation_errors {
+    color: $dark-orange;
+    background: $white;
+    border-color: $dark-orange;
+    box-shadow: none;
+    width: fit-content;
+
+    > h2 {
+      color: $dark-orange;
+    }
+  }
+
+  .gform_body .gform_fields .gfield {
+    .ginput_container_select,
+    .ginput_container_time {
+      position: relative;
+
+      &::after {
+        content: "";
+        position: absolute;
+        z-index: -1;
+        mask-image: url("../../images/chevron.svg");
+        mask-repeat: no-repeat;
+        mask-size: contain;
+        mask-position: center;
+        background-repeat: no-repeat;
+        background-color: currentColor;
+        transform: rotate(90deg);
+        width: 12px;
+        height: 12px;
+        right: $sp-2;
+        top: 19px;
+
+        html[dir="rtl"] & {
+          right: auto;
+          left: $sp-2;
+        }
+      }
+
+      select {
+        @extend .form-select;
+        background: transparent;
+        appearance: none;
+        position: relative;
+        min-width: 80px;
+      }
+    }
+
+    textarea,
+    input:not([type="file"]) {
+      @extend .form-control;
+    }
+
+    input {
+      height: 48px;
+    }
+
+    .gfield_checkbox,
+    .gfield_radio,
+    .ginput_container {
+      @extend .custom-control;
+
+      input[type="checkbox"] ~ label,
+      input[type="radio"] ~ label {
+        @extend .custom-control-description;
+        font-size: 1rem;
```

Comment:
Why `rem` instead of `px`?  I know that `rem` is better for responsive design but we're not following this rule (at all).

Reply:
Good point! We use a weird mix of `rem` and variables in [our other forms](https://github.com/greenpeace/planet4-master-theme/blob/master/assets%2Fsrc%2Fscss%2Flayout%2F_forms.scss) so I didn't really think it through and did something similar here... 😬 [Remove rem units](https://github.com/greenpeace/planet4-master-theme/pull/1638/commits/e30ee09092479ce754b669d18eec4f378707ce32)!